controllers/storagecluster/cephblockpools.go

```diff
@@ -44,6 +44,7 @@ func (r *StorageClusterReconciler) addPeerSecretsToCephBlockPool(initData *ocsv1
 func (r *StorageClusterReconciler) newCephBlockPoolInstances(initData *ocsv1.StorageCluster) ([]*cephv1.CephBlockPool, error) {
 	var mirroringSpec cephv1.MirroringSpec
 	poolName := generateNameForCephBlockPool(initData)
+	builtinMgrPoolName := "builtin-mgr"
```

Comment:
@malayparida2000 @travisn We'll use the pool name in the UI in order to filter out this pool and hide it in the UI, so we have to ensure that this name won't change over time.
cc @SanjalKatiyar

Reply:
1. Can we rely on some unique label which can be added to any pool which needs to be hidden on UI ??
2. Or, can we add a check on `Pool.Spec.Name === .mgr` for filtering in UI (assuming that it's less likely to get changed in future) ??

^ it might be okay to move forward with hard coded name as well (as for now only this pool needs to be hidden), so it's more of a suggestion/open-question.

Reply:
Let's go wit the more stable/reliable identifier: be it `metadata.name`, `spec.name` or another.

Reply:
As of now we have just this one pool which needs to be hidden so `metadata.name` sounds like a reasonable filter to me

Reply:
Then plz add a comment before the pool name explaining that name is used by UI... so that someone do not rename it 2-3 release from now by mistake or anything.

More generic approach (labels) is something we can revisit on need per basis in future, as UI can't keep on expanding list of hardcoded names.

Reply:
Added a comment.

Reply:
I think rook requires the name of the pool to be exactly this if I am not wrong https://github.com/rook/rook/blob/master/deploy/examples/pool-builtin-mgr.yaml

Reply:
The `metadata.name` of the pool is not strictly important for this pool. As a special pool that starts with a `.`, there is actually a `spec.name` field that specifies the required `.mgr` pool name. The `metadata.name` for the pool could be anything we want, but keeping it set to a constant `builtin-mgr` is my vote, for simplicity and consistency with upstream, since we want to hide from the UI anyway.